app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -276,9 +278,6 @@ export default {
             (sc) => sc.code === this.currentStatus.code
           ).statusCode;
           this.items = this.currentStatus.statusCodeDetail.nextCodes;
-
-          // Hiding REVISING status until backend can receive submitter's email.
-          this.items = this.items.filter((item) => item !== 'REVISING');
```

Comment:
We can't wholesale remove this -> we still need to remove Revising from the list if and only if the form does NOT allow users to create drafts. That should be a boolean somewhere in the form definition level. We'll need to make sure that is in place before enabling return to user flow.